frontend/src/app/Settings/Users/ManageUsers.tsx

```diff
@@ -228,7 +261,11 @@ const ManageUsers: React.FC<Props> = ({
         <Alert type="success" title={`User account removed for ${fullName}`} />
       );
       await getUsers();
+      // there is an unrelated bug here
+      // sortedUsers doesn't automatically update when getUsers() returns, so if you delete
+      // the first person in the sequence the active user is still the deleted user
       updateActiveUser(sortedUsers[0]); // arbitrarily pick the first user as the next active.
+      queryUserWithPermissions();
```

Comment:
Somehow this and the hooks interact and cause two requests to the graphql user(id) query.  The first one that gets will be canceled (and show up in red in the Chrome network tab).

Though it doesn't actually break anything functionally, I think we should probably fix it.

Reply:
Good catch, thanks! I suspect this was thanks to some asynchronous Javascript behavior, fixed by putting this logic into a `useEffect` below.